pycognaize/common/enums.py

```diff
@@ -73,6 +73,7 @@ class IqDocumentKeysEnum(enum.Enum):
     page = 'page'
     src = 'src'
     page_src = 'src'
+    text_span = 'text-span'
```

Comment:
Please add changelog in CHANGELOG.md file and change version in __init__.py file and create badges